examples/kuka_iiwa_with_orientation.py

```diff
@@ -0,0 +1,227 @@
+    return (env, goal)
+
+
+def set_planner(goal: GoalComposition, nr_obst: int = 0, degrees_of_freedom: int = 7):
+    """
+    Initializes the fabric planner for the panda robot.
+
+    This function defines the forward kinematics for collision avoidance,
+    and goal reaching. These components are fed into the fabrics planner.
+
+    In the top section of this function, an example for optional reconfiguration
+    can be found. Commented by default.
+
+    Params
+    ----------
+    goal: StaticSubGoal
+        The goal to the motion planning problem.
+    degrees_of_freedom: int
+        Degrees of freedom of the robot (default = 7)
+    """
+    absolute_path = os.path.dirname(os.path.abspath(__file__))
+    with open(absolute_path + "/iiwa7.urdf", "r", encoding="utf-8") as file:
+        urdf = file.read()
+    forward_kinematics = GenericURDFFk(
+        urdf,
+        rootLink="iiwa_link_0",
+        end_link="iiwa_link_ee",
+    )
+    planner = ParameterizedFabricPlanner(
+        degrees_of_freedom,
+        forward_kinematics,
+    )
+    collision_links = ["iiwa_link_3", "iiwa_link_4", "iiwa_link_5", "iiwa_link_6", "iiwa_link_7"]
+    iiwa_limits = [
+        [-2.96705973, 2.96705973],
+        [-2.0943951, 2.0943951],
+        [-2.96705973, 2.96705973],
+        [-2.0943951, 2.0943951],
+        [-2.96705973, 2.96705973],
+        [-2.0943951, 2.0943951],
+        [-3.05432619, 3.05432619],
+    ]
+    # The planner hides all the logic behind the function set_components.
+    planner.set_components(
+        collision_links=collision_links,
+        goal=goal,
+        number_obstacles=nr_obst,
+        number_plane_constraints=1,
+        limits=iiwa_limits,
+    )
+    planner.concretize()
+    return planner
+
+
+def run_panda_example(n_steps=5000, render=True):
+    nr_obst = 2
+    (env, goal) = initalize_environment(render, nr_obst=nr_obst)
+    planner = set_planner(goal, nr_obst)
+    # planner.export_as_c("planner.c")
+    action = np.zeros(7)
+    ob, *_ = env.step(action)
+    collision_radii = {3: 0.1, 4: 0.1, 5: 0.1, 6: 0.1, 7: 0.1}
+    # collision_links_nrs = [3, 4, 5, 6, 7]
+    for collision_link_nr in collision_radii.keys():
+        env.add_collision_link(0, collision_link_nr, shape_type='sphere', size=[0.10])
+
+    rot_matrix = np.array([[-0.339, -0.784306, -0.51956],
+                           [-0.0851341, 0.57557, -0.813309],
+                           [0.936926, -0.23148, -0.261889]])
+
+    # --- if you want to check if the orientation is correctly implemented: ---#
+    # rot_matrix = np.eye(3)
+    fk_full = planner._forward_kinematics.fk(q=planner._variables.position_variable(),
+                                             parent_link="iiwa_link_0",
+                                             child_link="iiwa_link_ee",
+                                             positionOnly=False)
+    fk_function = ca.Function("fk", [planner.variables._state_variables["q"]], [fk_full], ["q"], ["pose_ee"])
```

Comment:
you can directly access the fk with q being numpy as `fk.numpy`, see https://github.com/maxspahn/forwardKinematics/blob/bcbd5f3d5c67c28b8d72bd48496367d2a4cffdef/forwardkinematics/urdfFks/urdfFk.py#L78